presto-jdbc/src/test/java/io/prestosql/jdbc/TestJdbcResultSetTimezone.java

```diff
@@ -184,7 +185,7 @@ public void testTimestampWithTimeZone(Optional<String> sessionTimezoneId)
                 ImmutableList.of(
                         "TIMESTAMP WITH TIME ZONE '1970-01-01 00:00:00.000 +00:00'", // PostgreSQL
                         "from_tz(TIMESTAMP '1970-01-01 00:00:00.000', '+00:00')"), // Oracle
-                TIMESTAMP,
+                TIMESTAMP_WITH_TIMEZONE,
```

Comment:
With this there is another question. Should it still work with `TIMESTAMP` passed here. If so, should we have test coverage?

Reply:
It's just test's internal control flag to select which reference drivers to test against.
Whether TIMESTAMP or TIMESTAMP_WITH_TIMEZONE -- both PostgreSQL and Oracle are selected for comparison.
It has no further meaning.

Reply:
https://github.com/prestosql/presto/pull/6207